### PR TITLE
Drop repo_name column and update schema

### PR DESCRIPTION
**Pull request by MooreGuy**
repo_name is being replaced by repo since we want both the owner and
repo name. Code changes to stop referencing the repo_name happen in
https://github.com/iFixit/pulldasher/pull/83

### CODE REVIEW

**Comment by copperwall** [comment]
What about the schema :(

**Comment by lexahall** [comment]
QA ?

**Comment by MooreGuy** [comment]
@copperwall Yes! Thanks, I'll get that, also yeah, I'll have to think of QA for this, it touches a lot. dev_block :+1:

**Comment by MooreGuy** [comment]
un_dev_block :+1:

File: migrations/schema.sql [hunk 1]
```diff
@@ -140,7 +137,7 @@ CREATE TABLE IF NOT EXISTS `pulls` (
   `difficulty` int(11) DEFAULT NULL,
   PRIMARY KEY (`repo`,`number`),
   KEY `pulls_state` (`state`),
-  KEY `pulls_repo` (`repo_owner`,`repo_name`),
+  KEY `pulls_repo` (`repo_owner`),
```
Comment (nvaa36) [inline migrations/schema.sql]: It looks like the migration changes from the first one were not added. This is supposed to be a key on column `repo`, not `repo_owner`, right?

**Comment by lexahall** [comment]
@MooreGuy do you have any ideas for QA yet?

**Comment by cdcline** [comment]
CR 👍

**Comment by copperwall** [comment]
dev_block :star:  on https://github.com/iFixit/pulldasher/pull/92#discussion_r137600237

**Comment by MooreGuy** [comment]
I got slightly frustrated with splitting this into two pulls so this pull has all of these changes plus more: https://github.com/iFixit/pulldasher/pull/109

**Pull request closed without merging**